identity_document/src/error.rs

```diff
@@ -10,12 +10,12 @@ pub type Result<T, E = Error> = ::core::result::Result<T, E>;
 #[derive(Debug, thiserror::Error, strum::IntoStaticStr)]
 pub enum Error {
   /// Caused by errors from the [identity_core] crate.
-  #[error("{0}")]
+  #[error("unspecified failure")]
   CoreError(#[from] ::identity_core::Error),
-  #[error("{0}")]
+  #[error("unspecified failure")]
```

Comment:
This was done quickly in order to avoid duplicating messages when printing the whole error chain in the bindings.